### PR TITLE
Make the header functional, and update the content of index.html

### PR DESCRIPTION
**Pull request by jessdejong**
Add URLs to the header links to make them work,
update the content of index.html,
unify the font across the website,
delete the random greeting feature.

### CODE REVIEW

**Comment by estep-google** [comment]
It seems like maybe you haven't integrated earlier PRs here?

**Comment by jessdejong** [comment]
Hmm... I merged with master and called git pull but it seemed like it was already up to date? I believe this is up to date with the master branch.

File: portfolio/src/main/webapp/projects_experience.html [hunk 1]
```diff
@@ -4,32 +4,42 @@
...
         <a href="#" onclick="togglePopup()">Socials & Contact</a>
-        <span id="popup">
-          <a href="mailto:jessdejong843@gmail.com" target="_blank">Email</a>
-          <a href="https://linkedin.com/in/jess-dejong" target="_blank">LinkedIn</a>
-          <a href="https://instagram.com/jess.dejong" target="_blank">Instagram</a>
-          <a href="https://github.com/jessdejong" target="_blank">GitHub</a>
-        </span>
+      </li>
+      <li>
+        <ul id="popup">
```
Comment (daniel-google) [inline portfolio/src/main/webapp/projects_experience.html:27]: nit: same here

File: portfolio/src/main/webapp/hobbies.html [hunk 2]
```diff
@@ -4,20 +4,42 @@
...
+        <a href="projects_experience.html">Projects & Experience</a>
+      </li>
+      <li>
+        <a href="hobbies.html">Hobbies</a>
+      </li>
+      <li>
+        <a href="#" onclick="togglePopup()">Socials & Contact</a>
+      </li>
+      <li>
+        <ul id="popup">
```
[the remark this answers is not in the document]
Comment (jessdejong) [reply portfolio/src/main/webapp/hobbies.html:27]: I actually tried this, and it ended up messing with the layout of the header... leaving it in a separate list element was the formatting that I got to work. Should I still try and remove that last list element?

**Pull request merged**